### PR TITLE
[IMPROVEMENT] List Component

### PR DESCRIPTION
**Pull request by diegolmello**
## Proposed changes

Adds `List` component to be used.
Follow `List` Storybook to check usage.
It was applied on all places containing ListItem.

## How to test or reproduce

- Changed places should work/behave properly

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] Improvement (non-breaking change which improves a current function)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update (if none of the other choices apply)

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/RocketChat/Rocket.Chat/blob/develop/.github/CONTRIBUTING.md#contributing-to-rocketchat) doc
- [x] I have signed the [CLA](https://cla-assistant.io/RocketChat/Rocket.Chat.ReactNative)
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works (if applicable)
- [ ] I have added necessary documentation (if applicable)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/views/RoomActionsView/index.js [hunk 1]
```diff
@@ -667,117 +391,465 @@ class RoomActionsView extends React.Component {
...
+							size={50 * fontScale}
+							style={styles.avatar}
+							type={t}
+							baseUrl={baseUrl}
+							userId={user.id}
+							token={user.token}
+						>
+							{t === 'd' && member._id ? <Status style={sharedStyles.status} id={member._id} /> : null }
+						</Avatar>
+						<View style={[styles.roomTitleContainer, isGroupChat && styles.roomTitleDisabled]}>
```
Comment (djorkaeffalexandre) [inline app/views/RoomActionsView/index.js:436]: Let's just remove the chevron when the roomInfo is disabled instead of apply an opacity.

**Review by djorkaeffalexandre — CHANGES_REQUESTED** [review]

**Review by djorkaeffalexandre — APPROVED** [review]
LGTM

File: app/views/RoomActionsView/index.js [hunk 2]
```diff
@@ -667,117 +391,465 @@ class RoomActionsView extends React.Component {
...
 		}
-		if (!data.trailingSection) {
-			return <View style={[styles.sectionSeparatorBorder, { backgroundColor: themes[theme].auxiliaryBackground, borderColor: themes[theme].separatorColor }]} />;
+
+		if (t === 'd') {
+			return (
+				<List.Section>
+					<List.Separator />
+					<List.Item
+						title={`${ blocker ? 'Unblock' : 'Block' }_user`}
```
Comment (diegolmello) [inline app/views/RoomActionsView/index.js:532]: This icon doesn't exist

Comment (djorkaeffalexandre) [reply app/views/RoomActionsView/index.js:532]: It's working on this PR, looks like it just doesn't exists on the new icons selection updated on #2567.

**Pull request merged**